.github/workflows/main.yml

```diff
@@ -155,6 +155,49 @@ jobs:
           UBSAN_OPTIONS: print_stacktrace=1
         run: ${{ runner.workspace }}/b/ninja/simc output=/dev/null html=/dev/null json2=/dev/null json3=/dev/null ${{ matrix.simc_flags }}
 
+  ubuntu-clang-10-simc-tests:
+    name: simc-tests-${{ matrix.test }}-${{ matrix.tier }}-${{ matrix.fight_style }}-${{ matrix.class }}
```

Comment:
Nit: I'd reshuffle it a bit to `simc-tests-${{ matrix.test }}-${{ matrix.tier }}-${{ matrix.class }}-${{ matrix.fight_style }}` (move class before fight style) and maybe even drop `simc-`.

Not sure if it's my setup with dark theme styler or a general problem with github actions layout but it currently looks like this to me:
![image](https://user-images.githubusercontent.com/59478/94731330-42aec280-036d-11eb-867e-f2e62a1bd406.png)
which makes it really hard to find a specific per class run.

Reply:
Makes sense. I dropped the simc-tests- prefix as well, anything starting with the matrix.test name should be indication enough of what the job is doing.